storage/factory/storageServiceFactory.go

```diff
@@ -263,9 +272,18 @@ func (psf *StorageServiceFactory) createAndAddBaseStorageUnits(
 	shardId := core.GetShardIDString(psf.shardCoordinator.SelfId())
 	dbPath = psf.pathManager.PathForStatic(shardId, psf.generalConfig.StatusMetricsStorage.DB.FilePath)
 	statusMetricsDbConfig.FilePath = dbPath
+
+	dbConfigHandler = NewDBConfigHandler(psf.generalConfig.StatusMetricsStorage.DB)
```

Comment:
this function became too large. Maybe add a TODO to split it into smaller functions?

Reply:
added todo comment